src/olympia/shelves/models.py

```diff
@@ -2,25 +2,24 @@
 
 from olympia.amo.models import ModelBase
 
-SHELF_TYPES = ('categories', 'collections', 'extension',
-               'recommendations', 'search', 'theme')
+ENDPOINTS = ('collections', 'search')
 
-SHELF_TYPE_CHOICES = tuple((ty, ty) for ty in SHELF_TYPES)
+ENDPOINT_CHOICES = tuple((ty, ty) for ty in ENDPOINTS)
 
 
 class Shelf(ModelBase):
     title = models.CharField(max_length=200)
-    shelf_type = models.CharField(
-        max_length=200, choices=SHELF_TYPE_CHOICES, verbose_name='type')
+    endpoint = models.CharField(
+        max_length=200, choices=ENDPOINT_CHOICES)
     criteria = models.CharField(
         max_length=200,
-        help_text="e.g., ?recommended=true&sort=random&type=extension")
+        help_text='<i>?recommended=true&sort=random&type=extension</i>')
```

Comment:
The html isn't really necessary in the help_text - if we really want it italic we can do it in .css (I wouldn't bother though)